llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1948,6 +2047,26 @@ void AsmPrinter::emitFunctionBody() {
         break;
       }
 
+      // FIXME: Some indirect calls can get lowered to jump instructions,
+      // resulting in emitting labels for them. The extra information can
+      // be neglected while disassembling but still takes space in the binary.
+      if (TM.Options.EmitCallGraphSection && MI.isCall()) {
```

Comment:
Make sure there's a test that has a tail call that hits this too

Reply:
+1, I still don't see this...